Sources/Orbit/Components/SegmentedSwitch.swift

```diff
@@ -0,0 +1,157 @@
+import SwiftUI
+
+/// Showing two choices from which only one can be selected.
+///
+/// Currently SegmentedSwitch allows to have 2 segments.
+/// SegmentedSwitch can be in error state only in unselected state.
+///
+/// - Note: [Orbit definition](https://orbit.kiwi/components/interaction/segmentedswitch/)
+public struct SegmentedSwitch<Value>: View where Value: LosslessStringConvertible & Equatable {
```

Comment:
I don't remember such requirements on any other... "value" type. Is it really needed?

Reply:
SwiftUI seems to use `Identifiable` rather than `Equatable` in similar cases.

Reply:
`Equatable` is required in conditions like `if selection == secondOption` when component check what segment have to be highlighted. Another component using `LosslessStringConvertible` requirement is `InputField`